connector/apmconnector/transaction.go

```diff
@@ -355,9 +352,8 @@ func GetTransactionMetricName(span ptrace.Span) (string, TransactionType) {
 	if rpcService, rpcServicePresent := span.Attributes().Get("rpc.service"); rpcServicePresent {
 		if rpcMethod, rpcMethodPresent := span.Attributes().Get("rpc.method"); rpcMethodPresent {
 			return fmt.Sprintf("WebTransaction/rpc/%s/%s", rpcService.AsString(), rpcMethod.AsString()), WebTransactionType
-		} else {
-			return fmt.Sprintf("WebTransaction/rpc/%s", rpcService.AsString()), WebTransactionType
 		}
+		return fmt.Sprintf("WebTransaction/rpc/%s", rpcService.AsString()), WebTransactionType
```

Comment:
😆

Reply:
I didn't do this on a whim :) running `make all` breaks if I don't remove this `else` condition